library/tasks/SequenceDataWithMoleculeTagMetrics.wdl

```diff
@@ -50,7 +50,7 @@ task CalculateCellMetrics {
     File original_gtf
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.8"
+    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:kmk-mt-counts"
```

Comment:
We should be using a versioned release of docker images, not one with a branch name in it. Can this be updated before release?